### DISCUSSION TITLE
When do I set keybinds inside vscode vs neovim config?

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [vscode-neovim/vscode-neovim#1939] [Q&A] [answered]
I am still a bit puzzled as to how and when do I do something in vscode vs in neovim. For example setting keybindings, I can do it inside `settings.json` and inside `init.lua`

What I want to have is

1. `jk` to exit insert mode
2. `leader leader` to open file picker
3. `leader e` to open file explorer

And ideally something like harpoon plugin, can that work?

**Comment by theol0403** [comment, marked as answer]
Harpoon can't work. Insert mode bindings need to use composite keys. Anything else should generally be done in vim.

**Reply by Nikola-Milovic** [reply]
@theol0403 I see there's Harpoon for VSCode plugin, but I cannot get the mapping right. How do I let my neovim handle hotkeys like

`ctrl + e` `ctrl + n/s`?

I tried unbinding them, seems like `<c-t/n` works fine but `<c-n/s/e` just wont
